descriptors/ModuleDescriptor-template.json

```
@@ -116,7 +291,7 @@
           "permissionsRequired": ["inventory.items.item.put"],
           "modulePermissions": [
             "inventory-storage.items.item.get",
-            "inventory-storage.items.collection.get",
+            "inventory-storage.items.collections.get",
```

Please revert this.

---

Add 119 and remove 294, right?

---

Replace `collections` with `collection`.
```suggestion
            "inventory-storage.items.collection.get",
```